### ISSUE TITLE
Mix quote single symbol with inline tag syntax.

### ISSUE THREAD

**Issue by xgbuils** [jadejs/jade#2062]
Hi

I'm using jade 1.11.0 and I found this issue:
``` 
// test.jade
// ok
p a single quote ' symbol
// ok
p #[span inline tag syntax] 
// error
p mix #[span a single quote ' symbol and inline tag syntax]
```

``` bash
$ jade test.jade 

/usr/local/lib/node_modules/jade/lib/runtime.js:240
  throw err;
        ^
Error: test.jade:7
    5| p #[span inline tag syntax] 
    6| // error
  > 7| p mix #[span a single quote ' symbol and inline tag syntax]

The end of the string was reached with no closing bracket found.
    at parseMax (/usr/local/lib/node_modules/jade/node_modules/character-parser/index.js:26:13)
    at Parser.parseInlineTagsInText (/usr/local/lib/node_modules/jade/lib/parser.js:688:21)
    at Parser.parseText (/usr/local/lib/node_modules/jade/lib/parser.js:264:23)
    at Parser.tag (/usr/local/lib/node_modules/jade/lib/parser.js:811:29)
    at Parser.parseTag (/usr/local/lib/node_modules/jade/lib/parser.js:759:17)
    at Parser.parseExpr (/usr/local/lib/node_modules/jade/lib/parser.js:211:21)
    at Parser.parse (/usr/local/lib/node_modules/jade/lib/parser.js:122:25)
    at parse (/usr/local/lib/node_modules/jade/lib/index.js:104:21)
    at Object.exports.compile (/usr/local/lib/node_modules/jade/lib/index.js:205:16)
    at handleTemplateCache (/usr/local/lib/node_modules/jade/lib/index.js:174:25)
```

It seems a rare error because the first case works fine but third case doesn't.

**Comment by TimothyGu** [comment]
Duplicate of #1723 and fixed in the upcoming Jade 2.0.0.

**Comment by xgbuils** [comment]
Thanks!
